### ISSUE TITLE
Migrating from stable-5 to stable-6

### ISSUE THREAD

**Issue by Gitoffomalawn** [jacobalberty/unifi-docker#473]
General question: I'm still on the `stable-5` tag image, but looking to upgrade to `stable-6`. Is there anything else aside from updating my docker-compose file that I should look out for? Do I need to make a clean container and restore the config to it (and readopt everything), or will the persistent volume for version 5 work with version 6?

**Comment by ThomDietrich** [comment]
I would also be interested in peoples experience. I am currently on version 5.14.23

**Comment by Gitoffomalawn** [comment]
I finally braved the jump and moved from `stable-5` to `stable-6`. It was as easy as bringing the unifi containers down (`docker container stop <CONTAINER_ID>`), changing the image name in `docker-compose.yml` from `stable-5` to `stable-6`, and then raising everything back up with `docker-compose up -d`. 

There was no hassle. The version 5 volume was picked up by version 6 container without a hitch. There was no need to import a config, or readopt any hardware. After everything runs you can clean out the old containers and images. Just be prepared for everything to look different (although you can change back to the old interface if you want).

**Comment by CaptainStealthy** [comment]
Thanks for doing that, you gave me some more confidence, heh.

I'm running an older version of the 5.x channel, and updating straight to 6.5.55 didn't go too well (thank god for snapshots, LOL!) and the VM that I'm running it on is pretty old, too.

I've already spun up a new controller w/ docker-compose on 5.14.23, thanks to Jacob's awesome work. 😎 
My evil plan is to migrate my network to the new docker-ized controller, backup the data volumes, and then upgrade to v6.

**Comment by Gitoffomalawn** [comment]
Yeah, Jacob's docker version of the controller is great. I've been running with it ever since I started using Ubiquiti hardware. It's always worked flawlessly. The reason I was hesitant to move branches was mainly because of the horror stories I heard from people who migrated to the 6.0.x version of the controller. 

When those subsided, I decided it was time to switch, but was worried that the new version would be incompatible with the old files (some incompatibilities being the source of some of the stories I heard/read). But as usual, Jacob's stuff works like a charm.
